docs/differences_from_aws.md

```diff
@@ -0,0 +1,101 @@
+# Differences from AWS Architecture
+
+This document explains how GCP Ingestion differs from the AWS implementation.
+
+[AWS Data Platform Architecture](https://mana.mozilla.org/wiki/display/SVCOPS/Telemetry+-+Data+Pipeline+Architecture)
```

Comment:
This link looks like it's part of the table of contents. I'd move it to the sentence above and give it the text "AWS implementation (mozilla-internal diagram)"

Reply:
I did this instead, because it felt awkward to include a link description in the sentence:

> This document explains how GCP Ingestion differs from the [AWS Data Platform Architecture](https://mana.mozilla.org/wiki/display/SVCOPS/Telemetry+-+Data+Pipeline+Architecture)

added advantage being that link isn't guaranteed to be a diagram, jason just chose to put all the info into the diagram instead of more words.